### PR TITLE
GitHub Actions CI, bug fixes, and tests

### PR DESCRIPTION
**Pull request by phudtran**
This PR fixes and adds the following:

- Adds CI via GitHub Actions
- Removes Travis CI (Travis org shutting down)
- Fixes multiple cluster bring-up issues with NoneType Error
- Fix for load transit CLI unit tests
- Adds new single deploy YAML
- Adds starter code for additional validation of operator errors in E2E tests
- Temporarily disables the following tests due to regressions/CI issues
  - Unit Test: Network Policy CLI 
  - E2E Tests: Scaled EP test (Regression)
  - E2E Tests: Simple EP test (Fails in GitHub Actions but not locally, more debugging needed)

### CODE REVIEW

File: etc/deploy/deploy.mizar.yaml [hunk 1]
```diff
@@ -0,0 +1,492 @@
+---
```
Comment (vinaykul) [inline etc/deploy/deploy.mizar.yaml]: Does this replace deploy.mizar.components.yaml file?

File: mizar/obj/tests/test.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
```
Comment (vinaykul) [inline mizar/obj/tests/test.yaml:1]: :)

**Comment by vinaykul** [comment]
> * E2E Tests: Simple EP test (Fails in GitHub Actions but not locally, more debugging needed)

I took a quick scan and it looks good. Just one question about the failing action - will it block successful CI after this is merged?

/lgtm

**Comment by phudtran** [comment]
> > * E2E Tests: Simple EP test (Fails in GitHub Actions but not locally, more debugging needed)
> 
> I took a quick scan and it looks good. Just one question about the failing action - will it block successful CI after this is merged?
> 
> /lgtm

Nope, I've disabled these for now, so they won't run

File: etc/deploy/deploy.mizar.yaml (hunk 1 above)
Comment (phudtran) [reply etc/deploy/deploy.mizar.yaml]: Yes it does, thanks for the catch
I will remove the old file and version the new docker images accordingly

**Comment by vinaykul** [comment]
LGTM

**Review by Hong-Chang — APPROVED** [review]
LGTM

**Pull request merged**